tests/tests/test-staking/test-rewards-auto-compound.ts

```diff
@@ -342,7 +342,7 @@ describeDevMoonbeam("Staking - Rewards Auto-Compound - candidate leave", (contex
 
 describeDevMoonbeam("Staking - Rewards Auto-Compound - bottom delegation kick", (context) => {
   let newDelegator: KeyringPair;
-  let delegationCount = 0;
+  let delegationCount = 5;
```

Comment:
I'm not sure why this fixes this test. Is it possible that these txns aren't executing in order or something like that? (cc @nbaztec )

Reply:
I pushed the fix to your branch. It is very possible that the tx do not execute in order and since they come from different accounts using nonce is out of the equation.

Reply:
Thanks!